### ISSUE TITLE
Help on portainer install

### ISSUE THREAD

**Issue by trumpimar** [ghostfolio/ghostfolio#1251]
Hi,
I'm trying to install ghostfolio through portainer, but I'm running into an issue with the db user initialization. I'm following the instructions as stated by @LeonStoldt in https://github.com/ghostfolio/ghostfolio/issues/928#issuecomment-1133749694_
The postgres and redis containers start, but the ghostfolio container immediately stops because it can't connect to the postgres db. Looking at the logs for the postgres container it says: 
2022-09-11 08:13:19.158 UTC [32] FATAL:  password authentication failed for user "user"
2022-09-11 08:13:19.158 UTC [32] DETAIL:  Role "user" does not exist.
So is the POSTGRES_USER env variable not properly initialized when the stack is created or something? I'm lost.
I would appreciate any help.

**Comment by LeonStoldt** [comment]
Hi @trumpimar,
I just tested it out - doing it the same way as explained in the comment you mentioned - but I cannot reproduce your error.
I created a stack from git repository with the parameters mentioned in the comment and uploaded a .env file based on the provided [.env](https://github.com/ghostfolio/ghostfolio/blob/main/.env) from ghostfolio repository. The only thing I had to do after deploying the stack, was to restart the ghostfolio container as postgres wasn't ready to accept connections.

I will add an image of the stack I deployed, so you can compare it with your stack. It might help finding the issue.
![Screenshot from 2022-09-11 11-14-25](https://user-images.githubusercontent.com/31759512/189520116-97b79f94-b9b7-4880-8ebb-db0bf3b81f45.png)

I think your guess with the postgres user initialization seems promising. It seems that the role "user" was not created. If you want to correct it manually, you could dive into the postgres container and create the role "user" manually. It should be done by the "Quick Action" "Exec Console" in the "Containers" tab. As user to connect it should be `postgres` and you should be able to see a console with somthing like `postgres@75783424416c:`.
If everything worked correctly, you should be able to log in via `psql -U user -d ghostfolio-db` (-U = user | -d = database).
If the user initialization did not work and the default user ist still postgres, then you should be able to log in via psql (as user "postgres" is default). It will tell you via e.g. "failed: FATAL:  role "postgres" does not exist" if this user does not exist. You can create a user with something like this: [ref. StackOverflow Entry](https://stackoverflow.com/a/23934693).

Anyway, the easiest way would be to fix the stack deployment so you don't need to create users and databases manually if you are not familiar with it.

**Comment by trumpimar** [comment]
Hi @LeonStoldt. 
Thanks for the quick test on a Sunday. Using the console and psql I indeed determined that the user and the db hadn't been created. I tried to do it manually, but no joy either. 
Then I remembered that when I first tried I'd been mucking around with the env variables. Apparently my earlier attempt had screwed up the db init and because the volume wasn't deleted recreating the stack didn't solve that issue. So I started completely from scratch, also removing the volume before retrying. Now it works like a breeze. 

Thanks again for the pointers!
